src/composables/useGoalData.js

```diff
@@ -39,8 +39,7 @@ function getGoalDisplayName(category) {
  * @param {Object} options.apollo - Apollo client instance (optional, will use inject if not provided)
  * @returns Goal data and utilities
  */
-export default function useGoalData({ loans, apollo: apolloParam }) {
-	const apollo = apolloParam || inject('apollo');
+export default function useGoalData({ apollo }) {
```

Comment:
Why remove loans from props?

Reply:
Moved to `loadGoalData` function to use it in ATB modal watcher correctly.